resources/js/views/admin/BoundingBox.vue

```diff
@@ -216,7 +218,10 @@ export default {
         {
             if(this.activatedBox == -1)
             {
+                // Record the box which has been selected, and set all other boxes to be deselected:
+                for(var b = 0; b < this.boxes.length; b++) this.boxes[b].selected = false;
```

Comment:
Should use forEach for shorter code and readability 😄.

Reply:
thanks, done!